yarn-project/bb-prover/src/avm_proving.test.ts

```diff
@@ -64,10 +64,22 @@ const proveAndVerifyAvmTestContract = async (
   calldata = [functionSelector.toField(), ...calldata];
   const globals = GlobalVariables.empty();
   globals.timestamp = TIMESTAMP;
-  const environment = initExecutionEnvironment({ functionSelector, calldata, globals });
 
   const worldStateDB = mock<WorldStateDB>();
-  const contractInstance = new SerializableContractInstance({
+  // Top level contract call
```

Comment:
I can probably clean this all up in the future to be a single mocked bytecode since it all uses the `public_dispatch` function..